README.md

```diff
@@ -0,0 +1,22 @@
+# Arch manual pages
+
+# Dependencies
+
+- pyalpm
+- python-chardet
+- python-django
+- python-psycopg2
+- python-requests
+
+# Installation
+
+1. In the directory `mysite` copy `local_settings.py.example` to `local_settings.py` and modify it.
+Make sure to uncomment the appropriate database section (either sqlite or PostgreSQL)
```

Comment:
The `mysite/local_settings.py.example` does not contain any sqlite section. The code works only with `PostgreSQL` (due to `ArrayField` and `GinIndex`). Previously there were some conditionals to determine the database backend, I've just cleaned it up.